guide/source/program_administrator/opportunities_benefits.rst

```diff
@@ -39,119 +40,152 @@ of the opportunities for contractors include:
 
 * **Reduce Data Collection Burden for Program Participation.** Many of the
   HPXML-compliant software tools offer contractors a more streamlined modeling
-  or data collection experience.   The more that programs can align around both
-  the use of HPXML and the data set required from jurisdiction to jurisdiction,
-  the more makers of these tools can focus their investments to provide new and
-  innovative functionality to improve the contractor experience and reduce the
-  cost to use the tools.
+  or data collection experience. The more that programs can align around both
+  the use of HPXML and the data set required between program or utility
+  jurisdictions, the more that makers of these tools can focus their investments
+  to provide new and innovative functionality to improve the contractor
+  experience and reduce the cost to use the tools.  For example, contractors
+  participating Arizona Public Service's Home Performance with ENERGY STAR\
+  |reg| program, experienced on average a 31% reduction in administrative time
```

Comment:
@Gavinhastings I noticed this is the only place we use the trademark symbol when we're talking about Home Performance with ENERGY STAR. When I looked on [their website](https://www.energystar.gov/index.cfm?fuseaction=hpwes_profiles.showsplash), they are not using it. Should we remove it for consistency?